serverless/plugin/step_functions.json

```
@@ -1155,9 +1155,49 @@
         "type": "number",
         "minimum": 0
       },
+      "ItemBatcher": {
+        "type": ["object", "array"],
+        "MaxItemsPerBatch": {
+          "type": "number",
+          "minimum": 0
+        }
+      },
       "Iterator": {
         "$ref": "#/Aws_StateMachine_Definition"
       },
+      "ItemProcessor": {
+        "$ref": "#/Aws_StateMachine_Definition",
+        "ProcessorConfig": {
+          "type": ["object", "array"],
+          "Mode": {
+            "oneOf": [
+              {
+                "type": "string",
+                "enum": ["DISTRIBUTED"]
+              },
+              {
+                "type": "string",
+                "enum": ["INLINE"]
+              }
+            ]
```

If both are string, shouldn't this be a single enum instead of oneOf
```suggestion
            {
              "type": "string",
              "enum": ["DISTRIBUTED", "INLINE"]
            }
```